viv-wallet-app/src/services/login.ts

```diff
@@ -42,10 +42,4 @@ export class LoginService {
     getJwtToken(): string | undefined {
         return this.token;
```

Comment:
do not store the token, because it changes from time to time
we need to do something smarter here

Reply:
at least call getToken again everytime and assume permissions do not change, forcing user to relogging if things change (this should be acceptable for the MVP)

Reply:
I've just pushed a basic fix (call every time getToken). It will be enough for this PR and the MVP as well.